data.json

```
@@ -994,5 +994,11 @@
     "githubUsername": "bevennyamande",
     "imageUrl": "https://user-images.githubusercontent.com/10162710/120894576-6ff8f200-c619-11eb-9ee5-5078912d5550.png",
     "issueNumber": 552
-  }
+  },
+  {
+    "name": "Kamlesh Joshi",
+    "githubUsername": "kamleshjoshi8102",
+    "imageUrl": "https://user-images.githubusercontent.com/72374645/123034023-67b8f900-d406-11eb-829a-6df34ba91852.png",
+    "issueNumber": 558
+   }
```

```suggestion
  }
```
Think there is a space too much